UnityProject/Assets/Scripts/Core/Editor/ClearancePropertyDrawer.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using Systems.Clearance;
+using UnityEditor;
+using UnityEditor.Experimental.GraphView;
+using UnityEngine;
+
+namespace Core.Editor
+{
+	[CustomPropertyDrawer(typeof(Clearance))]
+	public class ClearancePropertyDrawer : PropertyDrawer
```

Comment:
owo wats dis

Reply:
![image](https://user-images.githubusercontent.com/56727168/183137721-14d877f9-4d18-4b26-a14a-a2743b2ac6cf.png)

Can now scroll and search for the clearance when picking it.

Reply:
oh that's neat as fuck